server.js

```diff
@@ -0,0 +1,41 @@
+const http = require('http');
+const fs = require('fs');
+const PORT = process.env.PORT || 8080;
+
+
+const server = http.createServer((request, response) => {
+  let method = request.method;
+  let url = request.url;
+
```

Comment:
You will need to add `/public` to the `uri` so that you can retrieve the underlying file, you really dont this outside of your readFile function. Keep the code, but put it in a better place.